galsim/base.py

```diff
@@ -1316,15 +1465,19 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
         if method == 'sb':
             gain *= local_wcs.pixelArea()
 
+        # Account for area and exptime.
+        prof *= area * exptime
+
         # Making a view of the image lets us change the center without messing up the original.
         imview = image.view()
         imview.setCenter(0,0)
 
         if method == 'phot':
             try:
-                image.added_flux = prof.SBProfile.drawShoot(
+                added_photons = prof.SBProfile.drawShoot(
                     imview.image, n_photons, uniform_deviate, gain, max_extra_noise,
                     poisson_flux, add_to_image)
+                image.added_flux = added_photons / area / exptime
```

Comment:
One thing I've been thinking over is that some of these new conventions for SEDs and flux units break some of the assumptions that people often make.  For example, the relationship between ```image.added_flux``` and the actual pixel values in the image is no longer trivial (unless ```area=1``` and ```exptime=1``` as is the default).  And, FWIW, I think different conventions out there might confuse people... e.g., in HST they renormalize the pixel values in many of the science images they distribute to an exptime of 1, so the actual exptime is not related to the pixel values in the way that we have done it here.  (I am not saying that what they do makes sense, just that it could be a cause for confusion.)  I wonder if there is an example one could give in the docstring, or added comments in some demos, that might be helpful?